### DISCUSSION TITLE
Solving problems in batch

### DISCUSSION THREAD

**Discussion by richardrl** [qpsolvers/qpsolvers#105] [Ideas]
Does this library support solving qps in batch? E.g. see this post: https://stackoverflow.com/questions/73076452/solving-many-convex-programs-in-parallel

**Comment by stephane-caron** [comment]
As of today (v2.1) qpsolvers does not provide this feature. But I think it would be an interesting feature to add :+1: I'd be happy to help work it into the library if you (or other contributors with a similar need) want to go ahead and propose a PR for it.
